src/spec-node/featuresCLI/test.ts

```diff
@@ -20,6 +20,7 @@ export function featuresTestOptions(y: Argv) {
 			'remote-user': { type: 'string', alias: 'u', describe: 'Remote user.  Not used for scenarios.', },  // TODO: Optionally replace 'scenario' configs with this value?
 			'log-level': { choices: ['info' as 'info', 'debug' as 'debug', 'trace' as 'trace'], default: 'info' as 'info', description: 'Log level.' },
 			'skip-image-metadata': { type: 'boolean', default: false, description: 'Skip applying image metadata to the resultant test image.  Image metadata is potentially appended to base images built with the dev container CLI' },
+			'preserve-test-containers': { type: 'boolean', default: false, description: 'Do not remove test containers after running tests.' },
```

Comment:
Also, can we check in either of https://github.com/devcontainers/cli/blob/e3d12b3157cc2ec74805d6764f67ad461d49ecc1/src/test/container-features/featuresCLICommands.test.ts#L25 tests, that the containers are getting cleaned up as expected.